dashboard/client/src/theme.ts

```diff
@@ -28,6 +28,17 @@ const basicTheme: ThemeOptions = {
       elevation: 0,
     },
   },
+  overrides: {
+    MuiTooltip: {
+      tooltip: {
+        fontSize: "0.75rem",
+        fontWeight: 400,
+      },
+      tooltipPlacementRight: {
+        margin: "0 8px",
```

Comment:
This change is not in effect. It is overridden by another value.
<img width="326" alt="Screen Shot 2022-12-15 at 10 32 31 PM" src="https://user-images.githubusercontent.com/9677264/208036986-96e5400a-2f81-4e4a-be3c-498455f843ad.png">